Lib/_strptime.py

```diff
@@ -565,6 +566,10 @@ def _strptime(data_string, format="%a %b %d %H:%M:%S %Y"):
             hour, minute, second,
             weekday, julian, tz, tzname, gmtoff), fraction, gmtoff_fraction
 
+date_specs = ('%a', '%A', '%b', '%B', '%c', '%d', '%j', '%m', '%U',
+              '%w', '%W', '%x', '%y', '%Y',)
+time_specs = ('%T', '%R', '%H', '%I', '%M', '%S', '%f', '%i', '%s',)
```

Comment:
I don't have strong opinions about this, but my intuition is that these should be `set` or `frozenset` rather than `tuple`. Are these tuples for performance reasons (I am not sure I know when exactly it's faster to use a set for "lookup membership" rather than a tuple or list).